data/tweaks/openssl-legacy-negotiation/enable.yml

```diff
@@ -0,0 +1,41 @@
+---
+# ansible playbook to apply OpenSSL legacy negotiation settings
+
+- name: Apply OpenSSL Legacy Negotiation Settings
+  hosts: localhost
+  connection: local
+  become: true
+  gather_facts: true
+  tasks:
+      - name: Apply legacy negotiation settings
+        ansible.builtin.copy:
+            dest: /etc/pki/tls/openssl.d/legacy-negotiation.cnf
+            mode: "0644"
+            content: |
+                [openssl_init]
```

Comment:
If possible, include a comment in the written config file indicating that it was generated by this tweak.